config-env-itest-opensource.yaml

```diff
@@ -35,10 +33,15 @@ schema_blacklist:
 module_env_config:
     - namespace: data_pipeline
       config:
+        load_schematizer_host_and_port_from_smartstack: False
```

Comment:
i had tried to remove this, but it was getting set to true from someplace, i am not sure how. Needs some investigation